tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/Elasticsearch/ElasticsearchExerciser.cs

```diff
@@ -60,6 +60,16 @@ public void SetClient(string clientType)
         [Transaction]
         public void SearchAsync() => _client.SearchAsync();
 
+        [LibraryMethod]
+        [Transaction]
+        [MethodImpl(MethodImplOptions.NoOptimization | MethodImplOptions.NoInlining)]
+        public void MultiSearch() => _client.MultiSearch();
+
+        [LibraryMethod]
+        [Transaction]
+        [MethodImpl(MethodImplOptions.NoOptimization | MethodImplOptions.NoInlining)]
+        public void MultiSearchAsync() => _client.MultiSearchAsync();
```

Comment:
I know we use this pattern a lot where we call an async method without awaiting it, but I don't understand why. Is it because the MFA console app doesn't know how to do async?

Reply:
I think that this pattern is part of the problem with at least some of the flaky tests.

Reply:
Looks like the pattern could be modified to use proper async and the MFA console app would handle it -- `DynamicMethodExecutor` checks to see if the return is of type `Task` and `.Wait()`s it if so. Looks like quite a few of the other "Exerciser" classes do this already - MySqlExerciser, RestSharpExerciser, etc.